### ISSUE TITLE
#-1252522683------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1833]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<img alt="Thumbnail" src="https://velocityautomate.mapsdevext.arcgis.com/sharing/rest/content/items/d4f81ae9d48741f795eeeafb200bfdfe/info/thumbnail/ago_downloaded.png?token=9RK8DY0qvaqYcPFrrqGyoYFDXDccEfRwGocAgWn1h2QHvqS27CYPWeoGfDDhiWbouvrpPtFqOCpMftXaKvLAbf1OV5jkMmHXEJBTesSLghowUJTJED12MDaAZVapBwz3WAcdFP-Wc9wsgCtU4WMYXsJHJ7GlyCp5JYdkqB24gAF2tf3oMafMeLGL9CUHZL7CXvfHqVznkdwFFa1N3tAw8KzQAMCR3t_5PacpF1wlYTvwz-iXw0Byi4YBGzGHMvQ0" class="sc-knuQbY hqWLHc">`

<strong>Element Location: </strong>.sc-knuQbY

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-1252522683_All page content should be contained by landmarks.png`
